### DISCUSSION TITLE
Authorization property is not being updated when a card payment is voided

### DISCUSSION THREAD

**Discussion by LavanyaNiki** [microsoft/Dynamics365Commerce.Solutions#232] [Q&A]
We use a third-party payment connector to process card payments for sales. To keep track of payment activities, we've set up an extension table to log every relevant action.

Currently, when processing successful card payments, we have created a trigger for "SaveTenderLineRequest" and "OnExecuted" method, we deserialize data from the preprocessedtenderline for authorization property and save it to our logs. 

However, when voiding a payment, we encounter a problem: although correct payment information from the hardware station is being passed, the authorization property doesn't get updated and still retains the old sale data. due to which we are having difficulties in tracking the information for sale and void in our Extension table.

Why is the authorization property not getting updated?

Could someone help me on, how to get the updated data on Authorization property?

**Comment by shahbazchishty** [comment]
Hi LavanyaNiki,

Have you tried creating a trigger for "VoidPaymentServiceRequest"? 
I believe you should get your void response properties in response.

**Reply by AbhiRaKas** [reply]
Hello [shahbazchishty],

Yes ,we have tried creating trigger for "VoidPaymentServiceRequest" but encountered the same outcome, the "authorization property" not getting updated.

**Comment by AbhiRaKas** [comment]
@shahbazchishty Could you please help me with solution ?
